bandit/plugins/yaml_load.py

```diff
@@ -52,7 +52,7 @@
 from bandit.core import test_properties as test
 
 
-@test.test_id('B506')
+@test.test_id('B612')
```

Comment:
The old ID of B506 also needs to continue to work, otherwise you would be breaking existing users either explicating including or excluding that test.